packages/itwinui-react/src/core/Select/Select.tsx

```diff
@@ -423,6 +425,7 @@ const CustomSelect = React.forwardRef((props, forwardedRef) => {
           aria-expanded={isOpen}
           aria-haspopup='listbox'
           aria-controls={`${uid}-menu`}
+          styleType={styleType}
```

Comment:
If `styleType` is not supported for `CustomSelect`, should we pass it to `<SelectButton>`? Or would just removing it out of `rest` be sufficient?